picoquic/quicctx.c

```diff
@@ -1206,7 +1206,7 @@ int picoquic_reset_cnx(picoquic_cnx_t* cnx, uint64_t current_time)
     }
 
     if (ret == 0) {
-        ret = picoquic_setup_initial_traffic_keys(cnx);
+        picoquic_setup_initial_traffic_keys(cnx);
     }
 
```

Comment:
I see why the analyzer protests, but I am uneasy about not testing the return value. If initialization fails, we should somehow fail the call. Instead of removing the reference to the return code here, we should replace line 1215:
```
    ret = picoquic_tlscontext_create(cnx->quic, cnx, current_time);
```
by
```
    if (ret == 0) {
        ret = picoquic_tlscontext_create(cnx->quic, cnx, current_time);
    }
```